template.rb

```diff
@@ -121,6 +121,10 @@ def render_file(filename)
             g.javascripts     false
             g.helper          false
           end
+          # Load AdminUser model
+          config.autoload_paths += %W(\#{config.root}/vendor/app/models)
+          config.autoload_paths += Dir["\#{config.root}/vendor/app/models/**/"]
```

Comment:
@jelaniwoods Can you say a bit about what the purpose of this line is, and/or what the difference is between this line and the prior line?

Reply:
Hmm apparently nothing! I was looking at [this stackoverflow answer](https://stackoverflow.com/a/17951538) on how to include a file everywhere in a rails app. I must've not restarted the server when testing since I didn't think one line was enough.